### PR TITLE
Code cleanup of Interaction module

### PR DESCRIPTION
**Pull request by jokoho48**
Code cleanup of Interaction module.

### CODE REVIEW

File: addons/interaction/functions/fnc_openMenuSelectUI.sqf [hunk 1]
```diff
@@ -41,7 +40,8 @@ _actions = [localize LSTRING(InteractionMenu), localize LSTRING(Interact)] call
             _forEachIndex
```
Comment (PabstMirror) [inline addons/interaction/functions/fnc_openMenuSelectUI.sqf]: uses _forEachIndex - needs to be `forEach`

Comment (jokoho48) [inline addons/interaction/functions/fnc_openMenuSelectUI.sqf]: oh miss that ok

**Comment by jokoho48** [comment]
Merge Conflicts Fixed

**Comment by PabstMirror** [comment]
Looks good

File: addons/interaction/functions/fnc_openSelectMenu.sqf [hunk 2]
```diff
@@ -26,24 +26,24 @@
...
+    GVAR(SelectCancel) = _selectCancel;
     buttonSetAction [8855, QUOTE( call GVAR(SelectCancel); )]; // Cancel
     buttonSetAction [8860, QUOTE( (call compile (lbData [ARR_2(8866, lbCurSel 8866)])) call GVAR(SelectAccept); )]; // Accept
     lbSetCurSel [8866, 0];
-}else{
-    PARAMS_1(_customActions);
-
+} else {
     private ["_count", "_action"];
+    params ["_customActions"];
```
Comment (esteldunedain) [inline addons/interaction/functions/fnc_openSelectMenu.sqf]: I believe this is covered on line 29 already

File: addons/interaction/functions/fnc_openSelectMenu.sqf [hunk 3]
```diff
@@ -26,24 +26,24 @@
 
 private["_action", "_count", "_customActions", "_i"];
```
Comment (esteldunedain) [inline addons/interaction/functions/fnc_openSelectMenu.sqf]: Should `_customActions` still be defined as private?

File: addons/interaction/functions/fnc_addPassengersActions.sqf [hunk 4]
```diff
@@ -50,6 +49,7 @@ _actions = [];
                 _unit
             ];
     };
-} forEach crew _vehicle;
+    true
+} count crew _vehicle;
 
 _actions
```
Comment (commy2) [inline addons/interaction/functions/fnc_addPassengersActions.sqf]: bookmark vvvvvvvvvvvvvvvvvv

**Comment by commy2** [comment]
Done in https://github.com/acemod/ACE3/pull/2635

**Pull request closed without merging**